package.json

```diff
@@ -16,15 +16,16 @@
     "lint": "vue-cli-service lint",
     "release": "standard-version --no-verify",
     "release:alpha": "standard-version --no-verify --prerelease alpha",
-    "install:local": "npm install file:../x/packages/x-components/empathyco-x-components-3.0.0-alpha.8.tgz --no-save",
+    "install:local": "npm install file:../x/packages/x-components/empathyco-x-components-3.0.0-alpha.10.tgz --no-save",
     "json:csv": "json-csv ./src/i18n/messages",
     "csv:json": "csv-json ./output ./src/i18n/messages"
   },
   "dependencies": {
     "@empathyco/x-adapter": "^6.2.0-alpha.4",
-    "@empathyco/x-components": "^3.0.0-alpha.9",
+    "@empathyco/x-components": "^3.0.0-alpha.10",
     "@empathyco/x-types": "^9.1.0-alpha.2",
     "reflect-metadata": "^0.1.13",
+    "tslib": "~2.3.0",
```

Comment:
Is TSLib mandatory if you use TypeScript to 4.3.5? Why are you adding this dependency now?

Reply:
It is not mandatory, but if we want to use some utils of Typescript, they are included in that library.